src/base/static/state/ducks/places.js

```diff
@@ -212,19 +212,13 @@ export default function reducer(state = INITIAL_STATE, action) {
     case UPDATE_PLACE:
       return {
         ...state,
-        placeModels: [
-          // Filter out the place model that's being updated.
-          ...state.placeModels.filter(
-            placeModel => placeModel.id !== action.payload.id,
-          ),
-          {
-            // Add it back in, along with the updated data.
-            ...state.placeModels.find(
-              placeModel => placeModel.id === action.payload.id,
-            ),
-            ...normalizeSubmissionSets(action.payload),
-          },
-        ],
+        placeModels: state.placeModels.map(placeModel => {
```

Comment:
I changed this around because I think this is the preferred way to update an item in an array. See: https://redux.js.org/recipes/structuring-reducers/immutable-update-patterns#updating-an-item-in-an-array

Reply:
Good call, this is much better :+1: